### DISCUSSION TITLE
Failed to get scan range, no queryable replica found in tablet（doris：Version : 0.15.0-rc04）

### DISCUSSION THREAD

**Discussion by TonyHowardPark** [apache/doris#9696] [RFC]
当进行了一个表自关联（表数据1亿条）
报错信息：Memory exceed limit. Hash join；Used: 2295220464, Limit: 2147483648. You can change the limit by session variable exec_mem_limit；
然后BE节点挂了，后面重启后，就报错“执行失败: Error Failed to execute sql:com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: errCode = 2, detailMessage = Failed to get scan range, no queryable replica found in tablet: 11432；”
找了好多资料，没找到解决方案，且没有复现出该问题。

**Comment by Ustinov-lwp** [comment]
同样的问题  我在1.2.6中也遇到了，现在找不到任何解决办法，求助
